films/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from films.views import FilmMainView
+
+urlpatterns = [
+    path('/main', FilmMainView.as_view()),
```

Comment:
현재 URI는 `/films/main`으로 되어 있습니다.
메인 페이지에서 보여주는 데이터라고 생각하셔서 main이라는 단어를 추가하신 것 같아요!

RESTful API 설계 방식으로 엔드포인트를 만드실 때는 항상 자원(데이터) 기준으로 만들어야 합니다.
그러므로 `/films/main` 보다는 `/films`가 더 적합해 보입니다~

```suggestion
    path('', FilmMainView.as_view()),
```

Reply:
반영하겠습니다. 감사합니다!